apache/datadog_checks/apache/data/conf.yaml.example

```diff
@@ -367,6 +367,12 @@ instances:
     #
     # empty_default_hostname: false
 
+    ## @param disable_generic_tags - boolean - optional - default: false
+    ## Generic tags such as `host` will be replaced by `apache_host` to avoid
+    ## getting mixed with other integraton tags.
+    #
+    disable_generic_tags: true
```

Comment:
Personally, I think the wording of `disable_generic_tags: true` isn't the most intuitive, is it possible to change to `enable_generic_tags: false`?

Reply:
we're using `disable_generic_tags` for a few integrations already (like snowflake), so let's keep this name